codalab/rest/account.py

```diff
@@ -22,66 +21,37 @@ def send_verification_key(username, email, key):
         recipient=email,
```

Comment:
Ugh, I broke this in one of my changes. Take a look at https://github.com/codalab/codalab-cli/pull/399